### ISSUE TITLE
Atom IDE and Tailwind3 : Observations after 1.1 update

### ISSUE THREAD

**Issue by sebilasse** [denoland/fresh#773]
Hello, 

I did the [1.1 update](https://deno.com/blog/fresh-1.1) 

Unfortunately I am using Github Atom with the [Atom IDE deno 1.26.0](https://atom.io/packages/atom-ide-deno) and writing .tsx
Now I get the error message about `JSX element implicitly has type any …` for each and every dom node: 
<img width="959" alt="Bildschirmfoto 2022-09-15 um 11 51 51" src="https://user-images.githubusercontent.com/750300/190374293-0d1bbbe9-ce6b-4180-be4a-ac2b6c1d63ab.png">

And I am using fresh with Tailwind3 and Tailwind3.1 (cause with really nice “themes” which I did before fresh) but now if I convert the twind plugin to the `next` branch [to avoid the tw() and just do it before render …], I am stuck at rehydrating.
See https://github.com/denoland/fresh/discussions/317 and for `twind@next` see 
https://github.com/tw-in-js/twind/issues/346#issuecomment-1173738004 ff. (it's awesome)

`twind@next` seems now to export the sheets in the main module [e.g. `virtual()`] 
Basically anything works. The problem is with rules (see https://github.com/tw-in-js/twind/blob/next/documentation/migration.md) at rehydration.
I can post an example with what I have.

**Comment by lucacasonato** [comment]
The current twind plugin is not compatible with twind@next. I'll have to publish a `twind-next` plugin for that.

**Comment by tzakharko** [comment]
Since twind@next just got a new release and is rapidly progressing it would be great if fresh had a plugin ready. I tried to look at it, but unfortunately I don't understand enough of either twind nor fresh to make the plugin work. Would be great if someone more knowledgeable had a look at it. Tailwind3 support along with other cool new things in twind@next would be a big improvement.

**Comment by marvinhagemeister** [comment]
Closing this as we've switched to Tailwind CSS which allows us to compile the style sheet ahead of time and move it out of the render path. The Tailwind CSS plugin will be released in the next Fresh 1.6 version which I'm hoping to release later this week.
